Gemfile.lock

```diff
@@ -356,6 +356,8 @@ GEM
     nokogiri (1.15.4)
       mini_portile2 (~> 2.8.2)
       racc (~> 1.4)
+    nokogiri (1.15.4-x86_64-linux)
+      racc (~> 1.4)
```

Comment:
Why are these changes?

Reply:
I was receiving the following error: Error: The process '/opt/hostedtoolcache/Ruby/3.1.2/x64/bin/bundle', so as a solution, I ran bundle lock --add-platform x86_64-linux

Reply:
What error were you receiving?

`Error: The process '/opt/hostedtoolcache/Ruby/3.1.2/x64/bin/bundle'` doesn't look like a complete error.
Anyway, I don't think you should add these changes in your PR,  as the error is probably due to a problem with your local environment, I guess.